Backpack/src/main/java/net/skyscanner/backpack/text/BpkTextField.kt

```diff
@@ -102,21 +103,36 @@ open class BpkTextField @JvmOverloads constructor(
     }
 
     this.iconTintColor = iconColor
-    setTextColor(textColor)
+    setTextColor(
+      colorStateList(
+        color = textColor,
+        pressedColor = textColor,
+        focusedColor = textColor,
+        activatedColor = textColor,
+        disabledColor = textColorDisabled,
+      )
+    )
     setHintTextColor(
-      ColorStateList(
-        arrayOf(intArrayOf(android.R.attr.state_focused), intArrayOf()),
-        intArrayOf(hintFocusedColor, hintNormalColor)
+      colorStateList(
+        color = hintNormalColor,
+        pressedColor = hintNormalColor,
+        focusedColor = hintFocusedColor,
+        activatedColor = hintFocusedColor,
+        disabledColor = textColorDisabled,
       )
     )
 
-    val padding = resources.getDimensionPixelSize(R.dimen.bpkSpacingMd) +
+    val paddingHorizontal = resources.getDimensionPixelSize(R.dimen.bpkSpacingBase)
```

Comment:
In the new design the spacings are different. It's usually 16dp horizontally and 14dp vertically.